tool/grind.dart

```diff
@@ -81,15 +81,94 @@ Future buildSdkDocs() async {
       sdkDocsDir.path, new Future.value(Directory.current.path));
 }
 
+class WarningsCollection {
+  final String tempDir;
+  final Map<String, int> _warningKeyCounts;
+  final String branch;
+  final String pubCachePath;
+  WarningsCollection(this.tempDir, this.pubCachePath, this.branch)
```

Comment:
was this dartfmt'd? I would have thought it would insert a newline between the field and the ctor.

Reply:
I did forget to run dartfmt, but it did not do that even when run.

Reply:
:disappointed: